### ISSUE TITLE
[UWP][Accessibility] Luminosity contrast ratio is less than 3:1 for outlines under 'FlightItinerary.JSON' list item

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3842]
# Platform

* UWP

# Version of SDK

master

# Issue

Repro Steps:​

1. Launch the application.​

2. New screen starts appearing. Navigate to Left pane JSON files.
3. Now select "FlightItinerary.JSON" button. It will update the Middle and Right Pane. Navigate all the elements of both the panes.

4. Open Color contrast Analyzer tool.
5. Select foreground and background color of outlines using color picker control of tool.
6. Verify the result of Color contrast Analyzer tool.

Actual Result:

Luminosity contrast ratio is less than 3:1 for outlines under FlightItinerary.JSON list item.

Expected Result:

Luminosity ratio should be greater than or equal to 3:1 for outlines.
